pysat/tests/test_instrument.py

```diff
@@ -2335,53 +2141,46 @@ def test_set_bounds_fname_with_frequency_and_width(self):
     @pytest.mark.parametrize("values", [('2009-01-01.nofile',
                                          dt.datetime(2009, 1, 1),
                                          '2009-01-03.nofile',
-                                         dt.datetime(2009, 1, 3),
-                                         2, 2),
+                                         dt.datetime(2009, 1, 3), 2, 2),
                                         ('2009-01-01.nofile',
                                          dt.datetime(2009, 1, 1),
                                          '2009-01-04.nofile',
-                                         dt.datetime(2009, 1, 4),
-                                         2, 3),
+                                         dt.datetime(2009, 1, 4), 2, 3),
                                         ('2009-01-01.nofile',
                                          dt.datetime(2009, 1, 1),
                                          '2009-01-05.nofile',
-                                         dt.datetime(2009, 1, 5),
-                                         3, 1)])
+                                         dt.datetime(2009, 1, 5), 3, 1)])
     def test_iterate_bounds_fname_with_frequency_and_width(self, values):
         """File iteration in bounds with step/width > 1, exclude stop bounds."""
 
         out = self.support_iter_evaluations(values, for_loop=True)
         # verify range of loaded data
```

Comment:
This particular comment is repeated throughout with lower case.

Reply:
Also should have an empty line between the last code line and the comment for legibility.